web/modules/custom/joinup_core/joinup_core.install

```diff
@@ -60,3 +62,94 @@ function joinup_core_requirements($phase) {
 function joinup_core_update_8100(&$sandbox) {
   \Drupal::service('module_installer')->install(['config_sync']);
 }
+
+/**
+ * Update the type of collections, solutions, releases and distributions.
+ *
+ * Two things will happen here.
+ * The first is that all related field mappings will be updated.
+ * The second is that all entities in the database will be updated.
+ */
+function joinup_core_update_8102() {
+  $entity_type_id = 'rdf_entity';
+  $query = Database::getConnection('default', 'sparql_default');
```

Comment:
Let's use a correct naming for this var. Query doesn't seem to describe its purpose. `$connection`, `$sparql_connection` or just `$sparql`?